### PR TITLE
Demonstrature

### PR DESCRIPTION
**Pull request by hsmajlovic**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,43 @@
+MANIFEST
```
Comment (hsmajlovic) [inline .gitignore]: Why do we need .gitignore here?

File: 1. Linearna regresija (poglavlje o regresionoj analizi).ipynb [hunk 2]
```diff
@@ -85,7 +85,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
```
Comment (hsmajlovic) [inline 1. Linearna regresija (poglavlje o regresionoj analizi).ipynb:88]: Curious. I assume all these changes are due to running your own instance of this .ipynb? Do you know maybe why this changes are caught anyway?

**Review by hsmajlovic — COMMENTED** [review]
Lgtm with some minor comments?

File: 1. Linearna regresija (poglavlje o regresionoj analizi).ipynb (hunk 2 above)
Comment (hsmajlovic) [reply 1. Linearna regresija (poglavlje o regresionoj analizi).ipynb:88]: Maybe you pushed a running instance of this particular .ipynb?

File: .gitignore (hunk 1 above)
Comment (saidsalihefendic) [reply .gitignore]: Mainly, it was due to prevent pushing .ipynb checkpoints from my local instance of jupyter notebook, that is why I added the .gitignore.

File: 1. Linearna regresija (poglavlje o regresionoj analizi).ipynb (hunk 2 above)
Comment (saidsalihefendic) [reply 1. Linearna regresija (poglavlje o regresionoj analizi).ipynb:88]: Yes, I pulled from the master branch and ran on the local instance. I have opened this particular notebook, but I may have forgotten to close & halt the notebook.
